catalog/spatial/csw/spatial-csw-transformer/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/TestCswTransformProvider.java

```diff
@@ -108,14 +108,16 @@ public void testMarshalOtherSchema() throws Exception {
         HierarchicalStreamWriter writer = new WstxDriver().createWriter(stringWriter);
         CswTransformProvider provider = new CswTransformProvider(mockMetacardManager, null);
         MarshallingContext context = new TreeMarshaller(writer, null, null);
-        context.put(CswConstants.OUTPUT_SCHEMA_PARAMETER, OTHER_SCHEMA);
+        context.put(CswConstants.TRANSFORMER_LOOKUP_KEY, TransformerManager.SCHEMA);
+        context.put(CswConstants.TRANSFORMER_LOOKUP_VALUE, OTHER_SCHEMA);
 
         ArgumentCaptor<String> captor = ArgumentCaptor.forClass(String.class);
 
         provider.marshal(getMetacard(), writer, context);
 
         // Verify the context arguments were set correctly
-        verify(mockMetacardManager, times(1)).getTransformerBySchema(captor.capture());
+        verify(mockMetacardManager, times(1)).getTransformerByProperty(captor.capture(),
+                captor.capture());
```

Comment:
We should validate against the schema since we are writing out XML by hand via XStream.